starport/services/scaffolder/init.go

```diff
@@ -31,15 +31,26 @@ var (
 // Init initializes a new app with name and given options.
 // path is the relative path to the scaffoled app.
 func (s *Scaffolder) Init(tracer *placeholder.Tracer, name string, noDefaultModule bool) (path string, err error) {
-	pathInfo, err := gomodulepath.Parse(name)
+	pwd, err := os.Getwd()
```

Comment:
I don't quite remember why we convert to abs path, can you please check without it to see if genny or other places in the code will have any problems with it?

If not, we can remove all this code related to converting to abs and rel.

Reply:
removed